scout/run.py

```diff
@@ -3792,6 +4027,12 @@ def finalize_outputs(
         for m in self.measures:
             # Set competed measure markets and savings and financial metrics
             mkts = m.markets[adopt_scheme]["competed"]["master_mseg"]
+            # Set shorthand for efficient-captured market data if these are
+            # available, and if they are not set shorthand to None
+            try:
+                eff_capt = mkts["energy"]["total"]["efficient-captured"]
+            except KeyError:
+                eff_capt = ""
```

Comment:
```suggestion
            eff_capt = mkts["energy"]["total"].get(["efficient-captured"], False)
```

Or have the return value be "" if a string is needed

Reply:
I think a string *is* actually needed here, so I've implemented accordingly.